llm_on_ray/inference/models/fuyu8b.yaml

```diff
@@ -13,15 +13,5 @@ ipex:
 model_description:
   model_id_or_path: adept/fuyu-8b
   tokenizer_name_or_path: adept/fuyu-8b
-  chat_processor: ChatModelwithImage
-  input_processor: FuyuProcessor
-  model_loader: FuyuForCausalLM
-  prompt:
-    intro: ''
-    human_id: '[INST] {msg} [/INST]
-
-      '
-    bot_id: ''
-    stop_words: []
-  config:
-    use_auth_token: ''
+  chat_model_with_image: true
+  chat_template: "{% if messages[0]['role'] == 'system' %}{% set loop_messages = messages[1:] %}{% set system_message = messages[0]['content'] %}{% else %}{% set loop_messages = messages %}{% set system_message = false %}{% endif %}{% for message in loop_messages %}{% if (message['role'] == 'user') != (loop.index0 % 2 == 0) %}{{ raise_exception('Conversation roles must alternate user/assistant/user/assistant/...') }}{% endif %}{% if loop.index0 == 0 and system_message != false %}{% set content = '<<SYS>>\\n' + system_message + '\\n<</SYS>>\\n\\n' + message['content'] %}{% else %}{% set content = message['content'] %}{% endif %}{% if message['role'] == 'user' %}{{ bos_token + '[INST] ' + content.strip() + ' [/INST]' }}{% elif message['role'] == 'assistant' %}{{ ' '  + content.strip() + ' ' + eos_token }}{% endif %}{% endfor %}"
```

Comment:
I'm not sure if this format is difficult for users to understand and configure. Is there a way to simplify this setup?

Reply:
I think I will create a new PR later and add Jinja usage instructions to the doc.